docs/reference/bpmn/script_tasks.md

```
@@ -187,4 +187,4 @@ Please see the [implementing files themselves](https://github.com/sartography/sp
 | times_executed_by_user                 | Returns the number of times the user has started an instance of the current process model.                                                                     |
 | user_has_started_instance              | Returns a boolean indicating if the user has started an instance of the current process model.                                                                 |
 | update_task_data_with_dictionary | Takes the value of a variable already present in task data and copies all of its key-value pairs into the root of task data. **Example**: update_task_data_with_dictionary(resp_order_start) will take all keys from resp_order_start and insert them into task data, allowing you to remove the original response variable.
-|
\ No newline at end of file
+|
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)